cwms-radar-model/src/main/java/mil/army/usace/hec/cwms/radar/client/model/TimeSeries.java

```diff
@@ -0,0 +1,256 @@
+/*
+ * Copyright (c) 2021
+ * United States Army Corps of Engineers - Hydrologic Engineering Center (USACE/HEC)
+ * All Rights Reserved.  USACE PROPRIETARY/CONFIDENTIAL.
+ * Source may not be released without written approval from HEC
```

Comment:
This should be the same as the license in the root of the project.

The last line there is especially... well, it's already public. (I assume this was just the boilerplate your IDE put in?)

Reply:
yeah this is just boilerplate, I'll update my IDE settings to match the license file.